jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -111,6 +116,12 @@ class SingleProcessingStore extends Reflux.Store {
   private isSubmissionLoaded = false;
   private isProcessingDataLoaded = false;
 
+  private displays = new Map<LanguageCode | StaticDisplays, boolean>([
```

Comment:
Please create a `/**` comment to describe the contents of this, especially what the boolean stands for